.github/workflows/ci.yaml

```diff
@@ -56,31 +55,42 @@ jobs:
           inv generateapidocs
           inv checkchanges --action="run inv generateapidocs"
 
-      - name: Build and save MetalLB images
-        run: |
-          inv build --binaries all
-          docker save quay.io/metallb/speaker:dev-amd64 > speaker.tar
-          docker save quay.io/metallb/controller:dev-amd64 > controller.tar
-          docker save quay.io/metallb/mirror-server:dev-amd64 > mirror-server.tar
+  build-test-images:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      matrix:
+        image: [speaker, controller, mirror-server]
```

Comment:
As before, I'd remove mirror-server (and add the newly added [configmaptocrs](https://github.com/metallb/metallb/tree/main/configmaptocrs) image

Reply:
I'll add a commit for configmaptocrs image. As for mirror-server, do you want to remove it entirely, or something else?

Reply:
See above